src/SFA.DAS.Notifications.Infrastructure/NotifyEmailService/NotifyClientWrapper.cs

```diff
@@ -58,7 +58,6 @@ private async Task SendMessage(NotifyMessage content, CommunicationType communic
             catch (NotifyClientException notifyClientException)
             {
                 Logger.Error(notifyClientException, $"Error sending communication {communicationType.ToString()} to Gov Notify with Gov.Notify Client");
-                throw;
```

Comment:
How do we know this has failed because of the phone number format?

Reply:
We don't know the reason for failure but the error is logged and NotifyClientException are filling the NServiceBus error queue. NotifyClientExceptions are used in the execution policy for retries from what I can see.